src/user/user.dtos.ts

```diff
@@ -0,0 +1,43 @@
+import { IsBoolean, IsEmail, IsMongoId } from 'class-validator';
```

Comment:
my gut tells me we will need to revisit user model after mvp (as we explore eth/ens login etc), but this can get us through for now